lib/puppet_x/chocolatey/chocolatey_version.rb

```diff
@@ -9,7 +9,7 @@ class ChocolateyVersion
 
       def self.version
         version = nil
-        choco_path = "#{PuppetX::Chocolatey::ChocolateyInstall.install_path}\\bin\\choco.exe"
+        choco_path = "#{PuppetX::Chocolatey::ChocolateyInstall.install_path}\\choco.exe"
```

Comment:
Arguably this is the one place this change is necessary, however I decided to make the change for the whole provider as well.